packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -394,6 +394,16 @@ class WebViewController {
     return _channel.invokeMethod("reload");
   }
 
+  /// Clears the resource cache.
+  ///
+  /// Note that the cache is per-application, so this will clear the cache for all WebViews used.
+  Future<void> clearCache() async {
```

Comment:
Looks like Android and iOS have some fine grained control over what exactly is being cleared.
I think it's ok to clear all (if we want to provide finer grained control we could add optional parameters here without it being a breaking change), but lets note in the documentation exactly what is being cleared on each platform.

Reply:
@amirh noted in the docs.